modules/health_quest/app/services/health_quest/resource/factory.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module HealthQuest
+  module Resource
+    class Factory
+      attr_reader :session_service, :user, :query, :resource_identifier, :options_builder
+
+      def self.manufacture(opts = {})
+        new(opts)
+      end
+
+      def initialize(opts)
+        @user = opts[:user]
+        @resource_identifier = opts[:resource_identifier]
+        @session_service = HealthQuest::Lighthouse::Session.build(user: user, api: opts[:api])
+        @query = Query.build(session_store: session_service.retrieve,
+                             api: opts[:api],
+                             resource_identifier: resource_identifier)
+        @options_builder = Shared::OptionsBuilder
+      end
+
+      def get(id) # rubocop:disable Rails/Delegate
```

Comment:
Do you need the rubocop comment here?

Reply:
Yes, I'm afraid so. My local rubocop fails if I don't have it in place. I wanted that method to be explicit since it's pretty crucial.